unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NavMap/NavmapView.cs

```diff
@@ -72,6 +72,7 @@ void Start()
             MapRenderer.OnParcelClicked += TriggerToast;
             MapRenderer.OnCursorFarFromParcel += CloseToast;
             CommonScriptableObjects.playerCoords.OnChange += UpdateCurrentSceneData;
+            DataStore.i.exploreV2.isOpen.OnChange += OnExploreChange;
```

Comment:
Should we unsubscribe this in `DataStore.i.exploreV2.isOpen.OnChange` or do you think it's not needed?

Reply:
Fixed, I was unsubbing from the wrong thing, my bad!